frontend/server/src/Controllers/Course.php

```diff
@@ -1919,7 +1919,7 @@ public static function apiRequests(\OmegaUp\Request $r): array {
      *
      * @omegaup-request-param string $course_alias
      * @omegaup-request-param bool $resolution
-     * @omegaup-request-param string $username
+     * @omegaup-request-param string $username     *
```

Comment:
```suggestion
     * @omegaup-request-param string $username
```

huh, quién sabe qué pasó aquí.